packages/@n8n/benchmark/src/scenario/scenarioDataLoader.ts

```diff
@@ -1,7 +1,7 @@
-import fs from 'node:fs';
-import path from 'node:path';
-import { Scenario } from '@/types/scenario';
-import { Workflow } from '@/n8nApiClient/n8nApiClient.types';
+import * as fs from 'node:fs';
+import * as path from 'node:path';
```

Comment:
Imports of these in cli do not need `*`?

Reply:
I swear there was some error/warning about those but can't reproduce it anymore. Will revert